### DISCUSSION TITLE
Passwordsafe export Vaultwarden import

### DISCUSSION THREAD

**Discussion by palicepta** [dani-garcia/vaultwarden#5021] [Ideas]
Hello,

We would like to export our passwords from the Passwordsafe (https://www.passwordsafe.com/en/) and import them to the Vaultwarden.

Please let us know if it's possible.

Thank you.

**Comment by stefan0xC** [comment]
I don't think there is specific importer provided by Bitwarden yet but [there's a common way ](https://bitwarden.com/help/condition-bitwarden-import/)that should work:
See https://bitwarden.com/help/import-from-passwordsafe/ for more information.

**Comment by palicepta** [comment]
Thanks a lot
